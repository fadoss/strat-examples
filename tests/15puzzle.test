games/15puzzle

srew puzzle2 using solve .
srew wikipedia using solve .
