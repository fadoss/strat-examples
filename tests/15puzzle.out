==========================================
srewrite in 15PUZZLE-MAIN : puzzle2 using solve .

Solution 1
rewrites: 28868
result Puzzle: 1	2	3	4 ;
5	6	7	8 ;
9	10	11	12 ;
13	14	15	b

No more solutions.
rewrites: 28868
==========================================
srewrite in 15PUZZLE-MAIN : wikipedia using solve .

Solution 1
rewrites: 40568
result Puzzle: 1	2	3	4 ;
6	5	7	8 ;
9	10	11	12 ;
13	14	15	b

No more solutions.
rewrites: 40568
Bye.
